src/test/shared/sam/debugger/samDebugConfigProvider.test.ts

```diff
@@ -3166,116 +2786,6 @@ Outputs:
             assertEqualLaunchConfigs(actualNoDebug, expectedNoDebug)
         })
 
-        it('debugconfig with extraneous env vars', async function () {
```

Comment:
This test doesn't seem to have a purpose anymore. 
Environment variables don't work with templates: https://github.com/aws/aws-toolkit-vscode/issues/2037